state/txindex/kv/kv.go

```diff
@@ -18,13 +18,12 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-const (
-	tagKeySeparator = "/"
-)
-
 var _ txindex.TxIndexer = (*TxIndex)(nil)
 
-// TxIndex is the simplest possible indexer, backed by key-value storage (levelDB).
+// TxIndex is the simplest possible indexer
+// It is backed by two kv stores:
+// 		1. txhash - result  (primary key)
```

Comment:
formatting is off

Reply:
Do you mean offsetting the numbered list. I did that intentionally :grimacing: . Do you think it's better aligned?

Reply:
yeah